docs/website/docs/overview/installation.md

```diff
@@ -270,8 +267,24 @@ type odo.exe.sha256
 
 4. Add the binary to your `PATH`
 
+## Visual Studio Code Plugin
```

Comment:
Since the VSCode Plugin currently installs odo v2, I was wondering if we shouldn't hide this section for now. When the VSCode Plugin migrates to v3, then we would consider adding this section back. Would even make more sense as part of the work to be done for https://github.com/redhat-developer/odo/issues/5674.
@kadel Any thoughts?

Reply:
Considering this doc points to v3, I agree with Armel. It makes more sense to hide this section for now.

Reply:
Brew also installs v2 of odo Mac, so 🤷 

Let's get @kadel 's opinion on this